code/game/machinery/bots/ed209bot.dm

```diff
@@ -288,16 +299,15 @@ Auto Patrol: []"},
 			continue
 
 		var/dst = get_dist(src, C)
-		if ( dst <= 1 || dst > 12)
+		if ( dst < 1 || dst > 12)
```

Comment:
I did this mostly because lasertag variants.

You can cheese them currently by standing next to them.